### PR TITLE
CF-615: remove nova from being marked as cloudfeeds:private

### PR DESCRIPTION
**Pull request by shintasmith**
Remove Nova from being marked ```cloudfeeds:private```. I did not find any trace of Glance eventTypes being marked as ```cloudfeeds:private```

### CODE REVIEW

**Comment by shintasmith** [comment]
But that doesn't make sense. The one that is marked with cloudfeeds:private is only the Nova usage events, which is not mixed format. The other ones we did not mark as ```cloudfeeds:private```

File: message_samples/nova/xml/cloudserversopenstack-nova-server-usage-v1-response.xml [hunk 1]
```diff
@@ -18,7 +18,6 @@
             xmlns:xsd="http://www.w3.org/2001/XMLSchema"
```
Comment (ChandraAddala) [inline message_samples/nova/xml/cloudserversopenstack-nova-server-usage-v1-response.xml]: I guess you have to remove this category from json examples as well (cloudserversopenstack-nova-server-usage-v1-response.xml)

**Comment by ChandraAddala** [comment]
Looks good to me.

**Comment by usnavi** [comment]
I just want to make sure that we don't do another observer role-remote-readd. ;-)

I don't understand how things are different in that they allow us to make nova events public now.

**Comment by shintasmith** [comment]
Ok, I'll go chase @pakazmir down.

**Comment by shintasmith** [comment]
@usnavi, I had @pakazmir rewrote and cleaned the story up. Check out:
https://jira.rax.io/browse/CF-615

and re-review this one. Thx!

**Pull request merged**